module/VuFind/src/VuFind/OAuth2/Repository/IdentityRepository.php

```diff
@@ -68,13 +68,14 @@ public function __construct(
     /**
      * Get a user entity by identifier.
      *
-     * @param int $identifier User ID
+     * @param int $identifier User Identifier
```

Comment:
I don't think int is the right type here anymore, since some identifier types might be strings.

Reply:
Fixed